frontend/webapp/app/page.tsx

```diff
@@ -24,11 +24,10 @@ export default function App() {
     switch (installation) {
       case CONFIG.NEW:
       case CONFIG.APPS_SELECTED:
-        router.push(`${ROUTES.SETUP}${state}`);
+        router.push(ROUTES.SETUP);
```

Comment:
Can you explain why the change? Looks like `state` above is currently unused and can be removed

Reply:
the state should be remain, i will fix it